src/libs/Middleware/SaveResponseInOnyx.js

```diff
@@ -15,6 +18,17 @@ function SaveResponseInOnyx(response, request) {
             return;
         }
 
+        // Supports both the old format and the new format
```

Comment:
We could possibly add some brief explanation of which one is the old vs new? 

NAB/just curious - How did things end up in this state? Is this temporary or are we going to support these two formats forever?

Reply:
Ah makes sense. I need to brush up on reliable updates.

Reply:
Thanks for the detailed explanation!